integration-tests/src/bin/read.rs

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use core::ffi::c_void;
+use libc::read;
+
+fn main() {
+    let buf = [0u8; 12];
+    unsafe {
+        read(0, buf.as_ptr() as *mut c_void, 12);
```

Comment:
Please use the comments on the write syscall test to update this to not use hardcoded values.